src/diffusers/pipelines/latent_consistency_models/pipeline_latent_consistency_text2img.py

```diff
@@ -590,6 +625,10 @@ def cross_attention_kwargs(self):
     def clip_skip(self):
         return self._clip_skip
 
+    @property
+    def do_classifier_free_guidance(self):
```

Comment:
ok :)

Reply:
just for consistency with the `# Copied from` style - it didn't make sense why we'd want to modify the `prepare_ip_adapter_image_embeds` method for one pipeline instead of reusing the same across all pipelines